devops/README.md

```diff
@@ -21,16 +21,16 @@ Amazon Linux 2 | x86_64 | [insiders-fast](#amazonlinux2-insiders-fast)
 Azure Linux 2 (mariner-2) | x86_64 | [insiders-fast](#azurelinux2-insiders-fast)
 CentOS 7 | x86_64 | [insiders-fast](#centos7-insiders-fast)
 CentOS 8 | x86_64 | [insiders-fast](#centos8-insiders-fast)
-Debian 10 | x86_64 / arm64 | [insiders-fast](#debian10-insiders-fast) / [prod](#debian10-prod)
-Debian 11 | x86_64 / arm64 | [insiders-fast](#debian11-insiders-fast) / [prod](#debian11-prod)
-Debian 12 | x86_64 / arm64 | [insiders-fast](#debian12-insiders-fast) / [prod](#debian12-prod)
+Debian 10 | x86_64 | [insiders-fast](#debian10-insiders-fast) / [prod](#debian10-prod)
```

Comment:
Were still previously releasing those packages and we will be shortly anyways. I wouldn't update the README given we will re-introduce once our rate-limits expire.

Reply:
We will re-add then. Until when we will readd publishing of ARM64 packages, we should not falsely advertise for them. Even for older ones that exist - and we do not want people to use unless they really know what they are doing, in which case documentation won't matter.  Do you agrere?

Reply:
Reverted these. We still have the part with "If you see any missing platforms or architectures that would help, please contact osconfigcore@microsoft.com."